client/src/components/Footer/index.tsx

```diff
@@ -27,163 +25,159 @@ function Footer(): JSX.Element {
           </p>
         </div>
         <div className='trending-guides'>
-          <h2 className='col-header'>{t('footer.trending-guides')}</h2>
-          <div
-            className='trending-guides-articles'
-            data-playwright-test-label='trending-guides-articles'
-          >
-            {/* the span elements were add so the link can be inline and its text would be the only active target */}
-            <span>
```

Comment:
will take a look at this again and let you know 👍🏻

Reply:
![Screenshot from 2023-11-16 17-14-36](https://github.com/freeCodeCamp/freeCodeCamp/assets/116457977/890a806a-ee52-4364-bad7-a9e25cfe3f17)

Items now rendered as shown in screenshot, with maximum of two columns instead of the previous three columns. Shall I remove the bullet points from the styling or raise as a separate issue?

Reply:
okay fixed the styling so it is now the same as previous UI, and across all device sizes - here's the screenshot
![Uploading Screenshot from 2023-11-20 10-22-45.png…]()